composer.json

```diff
@@ -28,6 +28,7 @@
     "automattic/vipwpcs": "^2.3",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "dg/bypass-finals": "1.1.3",
+    "fig-r/psr2r-sniffer": "^0.5.0",
```

Comment:
There might be more sniffs from this package that could be useful for us:

https://github.com/php-fig-rectified/psr2r-sniffer/blob/master/docs/sniffs.md#generic-21-sniffs

Especially the Slevomat ones (also included by this package) seem interesting:

https://github.com/slevomat/coding-standard#sniffs-included-in-this-standard

Some example config I just found:

https://github.com/wearerequired/coding-standards/blob/98326d69fab6d78808ec3d927fae6c870bbf9c91/Required/ruleset.xml#L106-L236